packages/core/src/helpers/getSchemaByResolvedExtensions.ts

```diff
@@ -75,6 +75,7 @@ export function getSchemaByResolvedExtensions(extensions: Extensions, editor?: E
           getExtensionField<NodeConfig['draggable']>(extension, 'draggable', context),
         ),
         code: callOrReturn(getExtensionField<NodeConfig['code']>(extension, 'code', context)),
+        whitespace: callOrReturn(getExtensionField<NodeConfig['whitespace']>(extension, 'whitespace', context)),
```

Comment:
whitespace was not being sent to the schema when it should have been, per: https://github.com/ueberdosis/tiptap/pull/4191#issuecomment-1767021524